yellowbrick/utils.py

```diff
@@ -155,6 +156,17 @@ def is_dataframe(obj):
 isdataframe = is_dataframe
 
 
+#From here: http://stackoverflow.com/questions/26248654/numpy-return-0-with-divide-by-zero
+def numpy_div0( a, b ):
```

Comment:
I converted this to div_safe, added some documentation to the function, and wrote tests demonstrating what the function can and can't do.